lib/application_layout_preloader.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationLayoutPreloader
   include ReadOnlyMixin
 
   def self.banner_json_cache
-    @banner_json_cache ||= DistributedCache.new("banner_json")
+    @banner_json_cache = DistributedCache.new("banner_json")
```

Comment:
Is this change intentional? A new distributed cache is being initialized each time the method is called now.